### ISSUE TITLE
[Accessibility] command bar buttons accessibility.

### ISSUE THREAD

**Issue by yzen** [devtools-html/debugger.html#4078]
Checklist:

- [x]  aria-label and title for each button are redundant. Just title will suffice.
- [x]  Enabled state should be accompanied but "aria-pressed" attribute (pause on exceptions for example) as there's no way for screen reader users to know when it's activated.
- [ ]  Improve focus styling. Right now dotted outline does not line up well with icons.

### Screenshot

![Screen Shot 2017-09-18 at 8.12.43 PM.png](https://shipusercontent.com/53cfb3cebfdb2dd7a21938d6e822f1ec/Screen%20Shot%202017-09-18%20at%208.12.43%20PM.png)

**Comment by jasonLaster** [comment]
#### Improve focus styling. Right now dotted outline does not line up well with icons.

```diff
diff --git a/src/components/SecondaryPanes/CommandBar.css b/src/components/SecondaryPanes/CommandBar.css
index 937443e..5a1ba39 100644
--- a/src/components/SecondaryPanes/CommandBar.css
+++ b/src/components/SecondaryPanes/CommandBar.css
@@ -49,6 +49,7 @@ html[dir="rtl"] .command-bar {

 .command-bar > button:focus {
   outline: none;
+  /* we're currently not doing anything... so we can do better :) */
 }
 ```
 
 #### aria-label and title for each button are redundant. Just title will suffice.
 
 ```diff
 diff --git a/src/components/SecondaryPanes/CommandBar.js b/src/components/SecondaryPanes/CommandBar.js
index 04ebd69..93f470e 100644
--- a/src/components/SecondaryPanes/CommandBar.js
+++ b/src/components/SecondaryPanes/CommandBar.js
@@ -71,7 +71,6 @@ function debugBtn(onClick, type, className, tooltip, disabled = false) {
   const props = {
     onClick,
     key: type,
-    “aria-label”: tooltip,
     title: tooltip,
     disabled
   };
```

**Comment by Garbee** [comment]
> what do you mean by enabled state? Is this something we do we/ css or JS?

So this is more for the "Pause on Exceptions" multi-state button. In the JS on activation/mod if any of it is "on" we'd have the `aria-pressed` attribute set. When it is turned off, remove the attribute. The rest of the buttons are completely transactional, so the attribute on those isn't needed as I understand things.

>  Improve focus styling. Right now dotted outline does not line up well with icons.

I completely agree. Styling needs to be improved in a few places for accessibility needs. However, this should be well thought through with a designer about the overall look and feel across the toolset. Making something up ad-hoc each time will lead to an inconsistent experience across the tools which isn't good. For now, the focus ring is the best we have until the DevTools design as a whole is thought over in detail.

**Comment by cvetanovskaa** [comment]
/claim Working on this issue with @michaelm396 @jacobT & @coledu

**Comment by lukaszsobek** [comment]
@yzen Is there still something left to do here or can the issue be closed?

**Comment by Garbee** [comment]
Styling of the button states needs to be handled still it appears to me.
